### DISCUSSION TITLE
Zod

### DISCUSSION THREAD

**Discussion by masterbater** [TypedProject/ts-express-decorators#2205] [Q&A]
How to use zod as a validation library

**Comment by Romakita** [comment]
Hello @masterbater 
I don’t encourage you to use another validator library because all framework is based on decorator that generate jsonschema for ajv validator, spec documentation for swagger ui and metadata for json-mapper.

but you can use another validator, look this page: https://tsed.io/docs/validation.html#custom-validation

see you
